nodes/HostBill/OperatonExecutor.ts

```diff
@@ -14,18 +16,15 @@ import { IDataObject, NodeApiError, NodeOperationError } from 'n8n-workflow';
 import { IHostBillApiCredentials } from '../../credentials/HostBillApi.credentials';
 import { INode, IOperation, IParam } from '../../generator/compactTypes';
 
-// tslint:disable-next-line: no-any
 export type Mappers = {[mapper: string]: (value: any) => string | undefined};
 
 type Dict<T> = {[key: string]: T};
 
 export const normalizeHost = (hostName: string) => hostName.replace(/\/$/, '');
 
-// tslint:disable-next-line: no-any
 const pick = (obj: any, props: string[]) => props.reduce((acc, i) => {
```

Comment:
Isn't it better if IDataObject is used instead of any?

Reply:
sure

Reply:
Could you change also the other occurrences of "any" to "IDataObject"?